server/routes.go

```diff
@@ -1566,6 +1567,24 @@ func (s *Server) ChatHandler(c *gin.Context) {
 	streamResponse(c, ch)
 }
 
+func (s *Server) version(w http.ResponseWriter, r *http.Request) {
+	if r.Method != http.MethodGet {
```

Comment:
we should also think about how we want to standardize on erroring throughout the API layer - we have the c.JSON error wrap in most places

Reply:
I think so too. However this 404s right now with a plaintext message and so I think it would be simplest to keep it a (better) plaintext error for now. Technically a body isn't required here, but in a follow up should return JSON errors to make it more obvious what's wrong